### ISSUE TITLE
Error in javascript console

### ISSUE THREAD

**Issue by melycastp** [creativetimofficial/paper-bootstrap-wizard#6]
Hi, when I was trying to implement this I got this error in the Javascript Console
```
Uncaught TypeError: $(...).validate is not a function
    at HTMLDocument.<anonymous> (paper-bootstrap-wizard.js:27)
    at i (jquery-2.2.4.min.js:2)
    at Object.fireWith [as resolveWith] (jquery-2.2.4.min.js:2)
    at Function.ready (jquery-2.2.4.min.js:2)
    at HTMLDocument.J (jquery-2.2.4.min.js:2)
```

**Comment by dragosct10** [comment]
Hi, @melycastp! Thank you for using our product. This error is appear for you because you call a function that you don't have. Did you have this function called in the scripts at the end of the html page? 

Regads, 
Dragos

**Comment by melycastp** [comment]
Yes, I have it. I call in the begining of the html page... Is that incorrect?

**Comment by melycastp** [comment]
I fix it

**Comment by melycastp** [comment]
Hi, it's me again. Where in the code I can edit the property of checkbox? I want to put a border on the image when it's selected. Thank you.

**Comment by dragosct10** [comment]
Hi, @melycastp! Did you mean those card-checkboxes? Regards, Dragos

**Comment by melycastp** [comment]
Yes, that ones

**Comment by dragosct10** [comment]
Hi, @melycastp! To change the card-checkboxes you must go in `assets/scss/paper-bootstrap-wizard/_card.scss` and there you will see the card-checkboxes. 

Regards, 
Dragos

**Comment by PHPadawan** [comment]
Hi @melycastp 
please tell me how did you fixed this?
